lib/src/components/NxProgressBar/NxProgressBar.scss

```
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @use "sass:math";
+
+.nx-progress-bar {
+  display: flex;
+  flex-direction: column;
+  gap: var(--nx-spacing-2x);
+  margin: var(--nx-spacing-6x) 0;
+
+  &--inline-counter {
+    flex-direction: row;
+
+    .nx-progress-bar__counter {
+      margin: 0;
+    }
+  }
+
+  &--inline {
+    display: inline-flex;
+    margin: 0 var(--nx-spacing-6x);
+    vertical-align: middle;
+  }
+
+  &--success {
+    color: var(--nx-swatch-green-25);
+  
+    .nx-progress-bar__counter {
+      border-color: var(--nx-swatch-green-25);
+      background-color: var(--nx-swatch-green-95);
+    }
+  }
+  
+  &--error {
+    color: var(--nx-swatch-red-40);
+  
+    .nx-progress-bar__counter {
+      border-color: var(--nx-swatch-red-40);
+      background-color: var(--nx-swatch-red-95);
+    }
+  }
+}
+
+.nx-progress-bar__counter {
+  margin: 0 var(--nx-spacing-2x) 0 0;
+}
+
+.nx-progress-bar__counter-and-label {
+  .nx-icon {
+    margin: 0 0 0 var(--nx-spacing-2x);
+    vertical-align: middle;
+  }
+}
+
+
+
+.nx-progress-bar__label-text {
+  margin-left: var(--nx-spacing-2x);
+  vertical-align: middle;
+}
+
+$normal-height: 25px;
```

This should be a local variable
```suggestion
$-normal-height: 25px;
```